tracer/src/Datadog.Trace/AppSec/SecurityCoordinator/HttpTransportBase.cs

```diff
@@ -0,0 +1,26 @@
+// <copyright file="HttpTransportBase.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.AppSec.Waf;
+using Datadog.Trace.Headers;
+
+namespace Datadog.Trace.AppSec;
```

Comment:
Doesn't match the folder name, maybe this class should be in the Transports folder?

Reply:
Ah, sorry, I see you've removed the transport folder, but the name space should still match the folder structure. Same comment for all the files in this folder.

```suggestion
namespace Datadog.Trace.AppSec.SecurityCoordinator;
```